psi4/driver/driver_cbs.py

```diff
@@ -1603,8 +2165,9 @@ def cbs(func, label, **kwargs):
         # Make energy(), etc. call
         response = func(molecule=molecule, **kwargs)
         if ptype == 'energy':
-            mc['f_energy'] = response
+            mc['f_energy'] = core.variable(VARH[mc['f_wfn']][mc['f_component']])
```

Comment:
why can't this be `response`? cleaner to keep things local rather than appealing to globals, esp. for ddd.

Reply:
Because `response` is always the `DFT TOTAL ENERGY` for a density functional, and we want the `component`. Would it be better to do `ret, wfn = func(..., return_wfn = True)` and then probe `wfn` for the component quantities?